streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -193,10 +195,25 @@
     <K1> KTable<K1, Long> count(KeyValueMapper<K, V, K1> selector,
                                 Serializer<K1> keySerializer,
                                 Serializer<V> valueSerializer,
-                                Serializer<Long> aggValueSerializer,
                                 Deserializer<K1> keyDeserializer,
                                 Deserializer<V> valueDeserializer,
-                                Deserializer<Long> aggValueDeserializer,
+                                String name);
+
+    /**
+     * Count number of records of this table by the selected key.
+     *
+     * @param selector the KeyValue mapper that select the aggregate key
```

Comment:
Question regarding naming consistency: Also, we perhaps want to rename "aggregate key" to "count key" or sth, given that we renamed e.g. `aggValueSerializer` to `countSerializer`?  (No strong opinion on this one.  If we assume developers will be used to "aggregate key" because that term is used for all the other functions where one needs to pass in a selector, we should keep the current terminology when talking about `count`.)